scripts/linux/install.sh

```diff
@@ -10,6 +10,12 @@ then
     exit -1
 fi
 
+if [ -n "$DSN_ROOT" -a -f "$DSN_ROOT/lib/libdsn.core.so" ]
```

Comment:
So how about we simply call ```make install``` when ```run.sh install``` is called? I've tried on my machine, and it seems ok...